addons/edi_stock/models/edi_move_cancellation_report_document.py

```diff
@@ -20,6 +20,13 @@ class StockMove(models.Model):
         index=True,
     )
 
+    cancelled_by = fields.Many2one("res.users", string="Cancelled By", readonly=True)
```

Comment:
Can we make this `cancelled_by_id`? Convention (although not always followed) is to end many2one fields with `_id`

Reply:
Changed